src/UI/Root/PrimaryPageManager.tsx

```diff
@@ -140,6 +153,16 @@ export class PrimaryPageManager implements PageManager {
   }
 
   getPages(): Page[] {
-    return Object.values(this.pageDictionary);
+    return Object.values(
+      features.instanceComposer
+        ? this.pageDictionary
+        : Object.fromEntries(
```

Comment:
Would you mind extracting this to a method to be more verbose as to the purpose of it?